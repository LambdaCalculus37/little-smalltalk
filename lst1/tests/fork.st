Class Main
[
	loop1
		10 timesRepeat: [17 print]
|
	loop2
		10 timesRepeat: [23 print]
|
	main
		[self loop1] fork.
		self loop2
]

