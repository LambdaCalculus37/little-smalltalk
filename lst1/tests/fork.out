Little Smalltalk
	17
23
17
23
17
23
17
23
17
23
17
23
17
23
17
23
17
23
17
23
Main
	

